build.sbt

```diff
@@ -31,7 +31,7 @@ lazy val release = Seq[ReleaseStep](
 
 inThisBuild(Seq(
   organization := "com.gu",
-  scalaVersion := "2.12.10",
+  scalaVersion := "2.12.12",
```

Comment:
everything still seems to build fine.  We should get onto 2.13 before scala 3 is released as the 2.13->3 migration can be automated, also the libraryDependencies are cross compatible 2.13<->3.
I think the main thing blocking us here was internal library dependencies e.g. event producer.